CMakeLists.txt

```diff
@@ -12,6 +12,30 @@ set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/defaults
                       ${CMAKE_SOURCE_DIR}/cmake/macros)
 
 include(Options)
+
+if(PXR_BUILD_AS_HOUDINI_PLUGIN)
```

Comment:
We should probably namespace this as RPR_BUILD_AS_HOUDINI_PLUGIN.  PXR-* is for pixar/USD

Reply:
I added `PXR_BUILD_AS_HOUDINI_PLUGIN` guided by `PXR_ENABLE_RIF_SUPPORT`. Perhaps, in that case, we should consider renaming this variable as well?

Reply:
Yes, that should be renamed as well...